streams/test-utils/src/main/java/org/apache/kafka/streams/test/ConsumerRecordFactory.java

```diff
@@ -153,20 +155,23 @@ public void advanceTimeMs(final long advanceMs) {
     }
 
     /**
-     * Create a {@link ConsumerRecord} with the given topic name, key, value, and timestamp.
+     * Create a {@link ConsumerRecord} with the given topic name, key, value, headers, and timestamp.
      * Does not auto advance internally tracked time.
      *
      * @param topicName the topic name
      * @param key the record key
      * @param value the record value
+     * @param headers the record headers
      * @param timestampMs the record timestamp
      * @return the generated {@link ConsumerRecord}
      */
     public ConsumerRecord<byte[], byte[]> create(final String topicName,
                                                  final K key,
                                                  final V value,
+                                                 final Headers headers,
                                                  final long timestampMs) {
         Objects.requireNonNull(topicName, "topicName cannot be null.");
+        Objects.requireNonNull(headers, "headers cannot be null.");
```

Comment:
Please add a test to `ConsumerRecordFactoryTest`

Reply:
it's added: `shouldNotAllowToCreateTopicWithNullHeaders`